pkg/plugin/v3/webhook.go

```diff
@@ -102,6 +110,27 @@ func (p *createWebhookPlugin) Validate() error {
 			" kind and version provided", p.commandName)
 	}
 
+	// if conversion flag is set but hub and spoke version is not provided, suggest it to user and return
+	if p.conversion && (p.spoke == "" && p.hub == "") {
+		klog.V(2).Info("Hub and Spoke version can be specified in the command to scaffold Convertible and Hub interfaces")
```

Comment:
Not worth a new dependency for this. Also, better if we move the message out of `Validate` (already did in the previous suggestion).
```suggestion
```